internal/gonut/cf/cf.go

```diff
@@ -235,6 +257,29 @@ func HasBuildpack(buildpackName string) (bool, error) {
 	return false, nil
 }
 
+// IsExternalBuildpack returns true if the buildpackName is a valid URL which returns
+// a 200 statuscode and links to a .zip file
+func IsExternalBuildpack(buildpackName string) (bool, error) {
+	//check if buildpackName is URL
+	if strings.HasPrefix(buildpackName, "http://") || strings.HasPrefix(buildpackName, "https://") {
```

Comment:
Rather than checking manually, I think it would be better to use https://golang.org/pkg/net/url/#Parse and try to parse the input string.

Reply:
I have tried to implement now but the Parse() function doesn't return an error in case of an invalid URL or simple string (usual buildpack name) and therefore only splits the URL in parts. I used now ParseRequestURI() for checking the validity of the URL but left the curl check and the .zip check to prevent CF errors and instead show custom errors.